pkg/runtime/scheme.go

```diff
@@ -90,6 +90,15 @@ func NewScheme() *Scheme {
 	}
 	s.converter = conversion.NewConverter(s.nameFunc)
 
+	// Add the deep copy funcs that can't auto-register themselves
+	// This should only be the runtime and unversioned packages
+	if err := s.AddGeneratedDeepCopyFuncs(GetGeneratedDeepCopyFuncs()...); err != nil {
+		panic(err)
+	}
+	if err := s.AddGeneratedDeepCopyFuncs(unversioned.GetGeneratedDeepCopyFuncs()...); err != nil {
```

Comment:
as you can see, no import was added... `runtime` already depends on `unversioned` for GroupVersion and GroupVersionKind types. I actually spent a long time trying to untangle that as part of this, and the ripples were... large. I'd be happy to add a TODO to reverse the direction and expose an unversioned.AddToScheme